spec.html

```diff
@@ -24459,7 +24585,7 @@ <h1>Runtime Semantics: PerformEval ( _x_, _callerRealm_, _strictCaller_, _direct
           1. If _strictEval_ is *true*, set _varEnv_ to _lexEnv_.
           1. If _runningContext_ is not already suspended, suspend _runningContext_.
           1. Let _evalContext_ be a new ECMAScript code execution context.
-          1. Set _evalContext_'s Function to *null*.
+          1. Set _evalContext_'s Function to _ctx_'s Function.
```

Comment:
It is necessary because we access this field here: https://github.com/tc39/ecma262/pull/1739/files#diff-3540caefa502006d8a33cb1385720803R8534

So eval code that creates a function should inherit the `"hide source"` and `"sensitive"` directives of the containing function:

```js
function f() {
  'hide source';
  return eval('(function g(){})').toString();
}
f(); // should be a string that matches NativeFunction
```

There may have been other spec bugs that were caused due to this field being set to `null`. I believe it was always meant to be inherited. But I haven't done an audit of all uses of the `Function` field of an execution context.

Reply:
Thanks for explaining.